e2e/app/page/workspace-base.ts

```diff
@@ -0,0 +1,98 @@
+import Link from 'app/element/link';
+import {buildXPath} from 'app/xpath-builders';
+import {ElementType} from 'app/xpath-options';
+import {Page} from 'puppeteer';
+import {waitWhileLoading} from 'utils/test-utils';
+import {waitForAttributeEquality} from 'utils/waits-utils';
+import AuthenticatedPage from './authenticated-page';
+
+export enum TabLabelAlias {
```

Comment:
nit: I think `TabLabels` is shorter and more descriptive. or `WorkspaceTabs`

Reply:
👍 Tabs also exists in Home page. `TabLabels` is better.